src/mlpack/tests/feedforward_network_2_test.cpp

```diff
@@ -65,7 +65,8 @@ TEST_CASE("RBFNetworkTest", "[FeedForwardNetworkTest]")
 {
   // Load the dataset.
   arma::mat trainData;
-  data::Load("thyroid_train.csv", trainData, true);
+  if (!data::Load("thyroid_train.csv", trainData, true)) 
```

Comment:
```suggestion
  if (!data::Load("thyroid_train.csv", trainData)) 
```
Third argument not required.